### ISSUE TITLE
Is it possible to have more than one HAP player in one scene with DKVFX ?

### ISSUE THREAD

**Issue by PockPocket** [keijiro/Dkvfx#15]
I'm trying to have multiple volumetric captures of different captures playing in one scene with DKVFX. Is that possible ?
So far I can only have one volumetric capture, when I add another HAP Player it can only play the same hap file.

**Comment by keijiro** [comment]
Could you provide a minimal project that reproduces the issue?

**Comment by PockPocket** [comment]
***@***.***D72FEE.79587080]

Each HAP player has a different HAP video file tied to it but when pressing play it plays the same file twice.

Here’s the project via google drive:
https://drive.google.com/drive/folders/1w_k4tJUHhCp69Gf6xHMZpsoyAbnic3Z-?usp=sharing

Thanks a lot for your help !

Mathieu.

**Comment by PockPocket** [comment]
![image](https://user-images.githubusercontent.com/73412598/114490034-21fbdb00-9be2-11eb-9420-8f55fc6d55e9.png)

**Comment by keijiro** [comment]
The converter objects write to the same set of render textures. You have to use different sets of render textures to use multiple VFX instances.

**Comment by keijiro** [comment]
I'm closing this issue now. Please feel free to reopen it for further problems.

By the way, your project was broken on my side, so I couldn't test it. Please try to make it minimal when sharing a project for issue reproduction.

**Comment by PockPocket** [comment]
Sorry for the broken project, I think the issues may be linked to plug-ins compatibility.

I've tried to duplicate the ASSET file Test.Metadata and place that new Test.Metadata_1 in the Metadata slot of the second clip converter but the issue still persists, it plays the same clip twice. I also tried to duplicate the position and color and place those new ones in the second converter but that also didn't worked. 

I'm wondering if duplicating is really the good move here, I didn't know how to create a new Metadata file so that's why I did it that way but maybe there's something I'm missing here ?

![image](https://user-images.githubusercontent.com/73412598/114569210-c6156e80-9c42-11eb-95c1-942926b15460.png)
